[BUG] Issues when running integration tests locally

### Is this issue already tracked somewhere, or is this a new report?

- [x] I've reviewed existing issues and couldn't find a duplicate for this problem.

### Current Behavior

When first running as per documentation `nox -s integration-tests`, getting 2 errors:

1 - `pytest.PytestCollectionWarning: cannot collect test class 'TestParam' because it has a __init__ constructor (from: tests/integration/test_cloud_download.py)`
2 - `ImportError while importing test module 'earthaccess\tests\integration\test_cloud_open.py'.ImportError: failed to find libmagic.  Check your installation`

First issue solved with giving specific parameter to ignore `pytest.PytestCollectionWarning` as I saw this warning ignored in dev pipeline:
`nox -s integration-tests -- -W ignore::pytest.PytestCollectionWarning`

Second issue is a platform-specific (Windows), I was able to solve it by installing DLL's package as per https://pypi.org/project/python-magic/ docs:
`pip install python-magic-bin`
That would allow to run via cmd, but with nox, had to change as well in `pyproject.toml`:
`python-magic >=0.4` to `python-magic-bin >= 0.4` which is not solution for other platforms, but maybe something worth adding to documentation for contributors.

Integration tests results:
`4 failed, 60 passed, 19 skipped, 26 warnings`
Failed errors are same for granules0 to 3 (seems like `fsspec` `remote_protocol` is being tuple, which is causing an issue):
```
__________________________________________________________________________________ test_open_virtual_mfdataset[granules3] ___________________________________________________________________________________ 

granules = [Collection: {'ShortName': 'M2T1NXSLV', 'Version': '5.12.4'}
Spatial coverage: {'HorizontalSpatialDomain': {'Geometry'...https://data.gesdisc.earthdata.nasa.gov/data/MERRA2/M2T1NXSLV.5.12.4/2025/01/MERRA2_400.tavg1_2d_slv_Nx.20250101.nc4']]

    def test_open_virtual_mfdataset(granules):
        # Simply check that the dmrpp can be found, parsed, and loaded. Actual parser result is checked in virtualizarr
>       vds = earthaccess.open_virtual_mfdataset(granules, concat_dim="time")
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
tests\integration\test_virtualizarr.py:33:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
earthaccess\dmrpp_zarr.py:188: in open_virtual_mfdataset
    vds = xr.open_dataset(
```
....
```

E               ValueError: Protocol not known: ('http', 'https')

.nox\integration-tests\Lib\site-packages\fsspec\registry.py:258: ValueError
```
  
Code snippet that causes the issue:
```
        storage_options = {
            "remote_protocol": fs.protocol,
            "remote_options": fs.storage_options,
        }
        vds = xr.open_dataset(
            str(ref_),
            engine="kerchunk",
            storage_options=storage_options,
        )
```


### Expected Behavior

Run `nox -s integration-tests` on a `main` branch gives no errors, all tests succeeding on any OS.

### Steps To Reproduce

1. Fetch main on Windows.
2. Setup conda environment.
3. Run `nox -s integration-tests`

### Environment

```markdown
- OS: Windows 11 Home, OS build 26200.6899
- Python: 3.14.0
```

### Additional Context

_No response_

--- a/tests/integration/param.py
+++ b/tests/integration/param.py
@@ -1,7 +1,7 @@
 from typing import TypedDict
 
 
-class TestParam(TypedDict):
+class ProviderParam(TypedDict):
     provider_name: str
 
     # How many of the top collections we will test, e.g. top 3 collections

--- a/tests/integration/test_cloud_download.py
+++ b/tests/integration/test_cloud_download.py
@@ -6,13 +6,13 @@ import earthaccess
 import pytest
 from earthaccess import Auth, DataGranules, Store
 
-from .param import TestParam
+from .param import ProviderParam
 from .sample import get_sample_granules, top_collections_for_provider
 
 logger = logging.getLogger(__name__)
 
 
-daac_list: list[TestParam] = [
+daac_list: list[ProviderParam] = [
     {
         "provider_name": "NSIDC_CPRD",
         "n_for_top_collections": 3,

--- a/tests/integration/test_cloud_open.py
+++ b/tests/integration/test_cloud_open.py
@@ -5,13 +5,13 @@ import magic
 import pytest
 from earthaccess import Auth, DataGranules, Store
 
-from .param import TestParam
+from .param import ProviderParam
 from .sample import get_sample_granules, top_collections_for_provider
 
 logger = logging.getLogger(__name__)
 
 
-daacs_list: list[TestParam] = [
+daacs_list: list[ProviderParam] = [
     {
         "provider_name": "NSIDC_CPRD",
         "n_for_top_collections": 2,

